tests/sb-util.test.ts

```diff
@@ -253,4 +253,9 @@ describe('Block class -------------------------', () => {
         const block = blocks.byId('8RrT-%*Y/x{|LPTvspp_');
         expect(block.prop('opcode')).toEqual('motion_changexby');
     });
+
+    test('can find parent block', () => {
+        const block = blocks.byId('8RrT-%*Y/x{|LPTvspp_');
+        expect(block.parent().prop('opcode')).toEqual('control_if');
+    });
```

Comment:
needs more tests - I'm curious how you are handling gaps in test coverage? 
It might be worth it to all least write-down test cases in a comment block for each PR with missing coverage. This is so we don't lose the intention if someone else is writing tests in the future, and we can get the benefit of unbiased tests but also a jump on where to start.

Reply:
One note: I will not be testing anything about the private internals of that storage yet (i'm not 100% its the long term best option) -- but I would happily add anything using the public exposed API right now

Could you reply a bit with "I'd like to see some tests covering ___ and ___ cases" and I'll enumerate some of them?

Reply:
thanks for the clarification - that explains a lot. My feedback was directed toward missing coverage of the new internal APIs. I’m curious to hear your feedback on my idea for adding comments to track intentionally missing tests cases for all the methods. If you think the internals are still very fluid, it's probably not worth it.